### PR TITLE
[simplify-networking] Refactor setup-ovs out of ci-tests

### PR DESCRIPTION
**Pull request by RamLavi**
As setup-ovs scneario is moving to kubernetes-nmstate [repo](https://github.com/nmstate/kubernetes-nmstate), the ci is changed to only test the ignition part.
Since this branch is not mature yet to be on coreos-ci and coreos-ci is still the preferred test-infra, we fork the relevant test to this repo and check them here. This will allow us to expand the test coverage should it be needed in the future.

### CODE REVIEW

File: tests/setup.sh [hunk 1]
```diff
@@ -4,3 +4,4 @@ set -ex
 
 dnf install -y git go make wget qemu qemu-img swtpm npm >/dev/null
 npm i -D tap-junit >/dev/null
+systemctl start docker
```
Comment (RamLavi) [inline tests/setup.sh:7]: memo-to-self: need to install golnag 1.16+

**Comment by kobi86** [comment]
Are we sure that we are moving to kubernetes-nmstate?

**Comment by RamLavi** [comment]
@kobi86 notice this is done on a separate branch.
The goal of this branch is to move to knmstate, as you formerly suggested.
It would be best to move from scripts to a formal api imo

**Comment by kobi86** [comment]
Its fine by me, but we had a problem with this in the past.
lets talk about this

**Comment by RamLavi** [comment]
> Its fine by me, but we had a problem with this in the past.
> lets talk about this 

Sure!

**Review by phoracek — APPROVED** [review]
Looks good as a base stage for the new approach.

**Comment by phoracek** [comment]
Feel free to merge this when ready. It is a development branch, so I don't think this should be waiting for the discussion to happen.

**Comment by RamLavi** [comment]
[updated](https://github.com/RHsyseng/rhcos-slb/compare/1ebd0816739a50f69d6aab03b5345e649d1d0e23..eee23cd6a9661625cf58cd90165dc5cdef3a25c7) setup.sh for fedora users, and fixed small typo

**Pull request merged**